week-4/Homework/mandatory/2-writers.js

```diff
@@ -39,6 +39,11 @@ let writers = [
   }
 ];
 
+
+writers.map(w => {
+  return console.log(`Hi, my name is ${w.firstName} ${w.lastName}.I am ${w.age} years old, and work as a ${w.occupation}.`)

```

Comment:
What does the `return` do here, and what's the difference between using `map` here (like you are), `filter` here, or `forEach` here?